### PR TITLE
Point search blog entry towards the manual

### PR DESCRIPTION
**Pull request by domcleal**

### CODE REVIEW

File: _posts/2012-01-01-advance-search-tips-in-foreman.md [hunk 1]
```diff
@@ -20,6 +20,9 @@ via the API, in a follow up blog I'll describe how to use the API and
 search via puppet directly as an alternative to storeconfigs -
 UPDATED: [here](http://blog.theforeman.org/2012/01/getting-foreman-search-results-into.html)
 
+**This blog entry has been updated and incorporated into our regular documentation.
+Please read on from the Foreman manual's [search chapter]({{site.baseurl}}/manuals/latest/index.html#4.1.5Searching).**
```
Comment (lzap) [inline _posts/2012-01-01-advance-search-tips-in-foreman.md:24]: Double-slash?

Comment (domcleal) [inline _posts/2012-01-01-advance-search-tips-in-foreman.md:24]: It was just changed in #471 to remove the trailing slash - so now we can put the / in here safely.  Perhaps long-term we should so we're less reliant upon `<base>`?

Comment (GregSutcliffe) [inline _posts/2012-01-01-advance-search-tips-in-foreman.md:24]: +1 for the slash, it seems to be standard Jekyll style.

**Comment by GregSutcliffe** [comment]
Merged as f167b4a, thanks.

**Pull request closed without merging**